packages/foam-vscode/src/features/preview-navigation.ts

```diff
@@ -25,6 +27,47 @@ const feature: FoamFeature = {
   },
 };
 
+export const markdownItWithNoteInclusion = (
+  md: markdownit,
+  workspace: FoamWorkspace
+) => {
+  return md.use(markdownItRegex, {
+    name: 'include-notes',
+    regex: /!\[\[([^\[\]]+?)\]\]/,
```

Comment:
let's make the linter happy, either by removing the escape char if really unnecessary, or by placing an es-lint directive to ignore the line